Task70/YandexScreenshot.cs

```diff
@@ -0,0 +1,35 @@
+﻿using NUnit.Framework;
+using System.Threading;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium.Support.UI;
+using System;
+using System.Collections.Generic;
+
+namespace Task70
+{
+    public class YandexScreenshot
+    {
+        protected IWebDriver _driver;
+
+        public YandexScreenshot(IWebDriver driver)
+        {
+            _driver = driver;
+        }
+        public void TakeScreenshot()
+        {
+            try
+            {
+                var screenshotDriver = _driver as ITakesScreenshot;
+
+                Screenshot screenshot = ((ITakesScreenshot) _driver).GetScreenshot();
+                screenshot.SaveAsFile(@"D:\Screenshots\SeleniumTestingScreenshot.jpg", ScreenshotImageFormat.Jpeg);
```

Comment:
Please, use your current directory project like one of these ways - [Tip](https://stackoverflow.com/questions/816566/how-do-you-get-the-current-project-directory-from-c-sharp-code-when-creating-a-c). This hardcoded path isn't good point:) Let's imagine that I will run your tests locally against my laptop which doesn't have disk D. In this case the TakeScreenshot method will fail because of disk D doesn't exist. I'm strongly recommend you to avoid using hardcoded paths in your future projects:)